### DISCUSSION TITLE
Feature request: Schedule overview

### DISCUSSION THREAD

**Discussion by tomh4** [pluginpal/strapi-plugin-publisher#24] [Ideas]
It would be very handy to have a list or even a calendar with all the published content at one glance.
Is there something like this planned?

**Comment by ComfortablyCoding** [comment]
A list overview of records scheduled to be published/unpublished as well as published/unpublished is planned. It will be a list over a calender as the design system does not have a calender compo at this time.

I have not gotten the chance to work on it yet. Will see if i can do it over the weekend.

**Comment by tomh4** [comment]
Great!
I might help, but I need to see if I find some time :)

**Comment by sqlninja** [comment]
Did anything ever come of this?
I am trying to simply figure how to show (and ultimately sort by) the publishAt field

**Reply by ComfortablyCoding** [reply]
Unfortunately no, nothing yet. Haven't had a chance to work on it.
